react-app/ccf/src/functions/storage.ts

```diff
@@ -0,0 +1,39 @@
+import { ref, uploadBytes, getDownloadURL, listAll } from "firebase/storage";
+import { storage } from '../index'
+
+export const handleUpload = async (file: File): Promise<string> => {
+    if (!file) {
+      throw new Error('No file selected');
+    }
+  
+    const storageRef = ref(storage, 'pdfs/' + file.name);
+  
+    try {
+      const snapshot = await uploadBytes(storageRef, file);
+      console.log('Uploaded a blob or file!');
+      
+      const downloadURL = await getDownloadURL(snapshot.ref);
+      console.log('File available at', downloadURL);
+      
+      return 'File uploaded successfully!';
+    } catch (error) {
+      console.error('Upload failed', error);
+      throw error;
+    }
+  };
+
+  export const listAndDownloadPDFs = async (): Promise<Array<{ name: string, url: string }>> => {
```

Comment:
We should separate the functionality of getting all pdfs and getting a single pdf. Applicants or reviewers won't need to get every pdf, but an admin account will.